Sources/Valkey/Commands/GeoCommands.swift

```diff
@@ -1030,15 +1073,57 @@ public struct GEOSEARCHSTORE: ValkeyCommand {
             unit.encode(into: &commandEncoder)
         }
     }
+    public struct ByPolygonVertices: RESPRenderable, Sendable, Hashable {
+        @usableFromInline let longitude: Double
+        @usableFromInline let latitude: Double
+
+        @inlinable public init(longitude: Double, latitude: Double) {
+            self.longitude = longitude
+            self.latitude = latitude
+        }
+
+        @inlinable
+        public var respEntries: Int {
+            longitude.respEntries + latitude.respEntries
+        }
+
+        @inlinable
+        public func encode(into commandEncoder: inout ValkeyCommandEncoder) {
+            longitude.encode(into: &commandEncoder)
+            latitude.encode(into: &commandEncoder)
+        }
+    }
+    public struct ByPolygon: RESPRenderable, Sendable, Hashable {
+        @usableFromInline let numVertices: Int
+        @usableFromInline let vertices: [ByPolygonVertices]
+
+        @inlinable public init(numVertices: Int, vertices: [ByPolygonVertices]) {
+            self.numVertices = numVertices
+            self.vertices = vertices
+        }
+
+        @inlinable
+        public var respEntries: Int {
+            RESPWithToken("BYPOLYGON", numVertices).respEntries + vertices.respEntries
+        }
+
+        @inlinable
+        public func encode(into commandEncoder: inout ValkeyCommandEncoder) {
```

Comment:
Can we make the placement of `@inlinable`/`@usableFromInline` consistent? I like the above the function call/variable definition version most. Follow up PR is fine.